doc/user/content/sql/create-connection.md

```diff
@@ -312,7 +312,7 @@ CREATE CONNECTION kafka_connection TO KAFKA (
 
 {{< tab "AWS IAM">}}
 
-{{< private-preview />}}
+{{< public-preview />}}
```

Comment:
Elsewhere on the page, we state public preview (ditto public preview in notion doc).